docs/Theming/Colors-And-Palettes.md

```diff
@@ -0,0 +1,11 @@
+# Colors & Palettes
+
+[Back to Theming](./../Theming.md)
```

Comment:
IMO, I'd prefer not nesting folders at all (at least for now). I want to avoid that so that in the future we can more easily migrate to other system. Just call this file `Theming-Colors-and-Palletes.md` and put in the main folder.